Bulldozer.FellowshipOne/Maps/Financial.cs

```diff
@@ -431,7 +432,12 @@ private void MapContribution( IQueryable<Row> tableData, long totalRows = 0 )
                         var parentAccount = accountList.FirstOrDefault( a => !a.CampusId.HasValue && a.Name.Equals( fundName.Truncate( 50 ), StringComparison.OrdinalIgnoreCase ) );
                         if ( parentAccount == null )
                         {
-                            parentAccount = AddFinancialAccount( lookupContext, fundName, $"{fundName} imported {ImportDateTime}", fundGLAccount, null, null, isFundActive.AsBooleanOrNull(), receivedDate, fundName.RemoveSpecialCharacters() );
+                            int? accountTypeDefinedTypeId = null;
+                            if ( !string.IsNullOrWhiteSpace ( fundType ) )
```

Comment:
code formatting issue:
```suggestion
                            if ( !string.IsNullOrWhiteSpace( fundType ) )
```